lib/platform/git/storage.ts

```diff
@@ -183,7 +184,13 @@ export class Storage {
       const cloneStart = process.hrtime();
       try {
         // clone only the default branch
-        await this._git.clone(config.url, '.', ['--depth=2']);
+        let opts = ['--depth=2'];
```

Comment:
I know we use `--depth=2` all over Renovate, but I'm not sure if it's actually useful? I think we could drop this to 1.

Reply:
I don't really know the system well enough, to weigh in on that. Probably more a question for people that know renovate better than me (like @viceice @rarkins )?